Externals/expr/include/expr.h

```diff
@@ -468,6 +479,17 @@ extern "C" {
       }
       return i;
     }
+    else if (c == '"') {
+      i++;
+      while (i < len && s[i] != '"')
+        i++;
+
+      if (s[i] != '"' || i <= 1)
+        return -5;
```

Comment:
Should be a unique error code with a comment:
`return -7; // missing expected quote`